scripts/momenihin.js

```diff
@@ -0,0 +1,19 @@
+/** A javascript program for HNGi7 task that outputs a string containing 
+the fullname, HNGi7 ID and language for HNG INTERNSHIP **/
+
+const myFullName = "Nwankwo Henry Ifunanya";
+const myHNGi7ID = "HNG-03667";
+const myLanguage = "Javascript";
+
+
+function helloWorld(fullName, ID, language){
+
+
+    const fullText = `Hello World, this is ${fullName} with HNGi7 ID ${ID} using ${language} for stage 2 task`;
```

Comment:
Add your email. to the fulltext. check the channel for the updated format

Reply:
I have added an email and have modified my code